### PR TITLE
WAC510 fix built-in script and wifi MAC addresses

### PR DESCRIPTION
**Pull request by 7604timo**
Fix macaddr_add function to work with sh/ash shells.
Properly set MAC addresses of wifi radios.

### CODE REVIEW

File: package/base-files/files/lib/functions/system.sh [hunk 1]
```diff
@@ -76,7 +76,8 @@ macaddr_add() {
 	local oui=${mac%:*:*:*}
 	local nic=${mac#*:*:*:}
 
-	nic=$(printf "%06x" $((0x${nic//:/} + $val & 0xffffff)) | sed 's/^\(.\{2\}\)\(.\{2\}\)\(.\{2\}\)/\1:\2:\3/')
+	nic=`echo $nic | sed 's/\://g'`
```
Comment (robimarko) [inline package/base-files/files/lib/functions/system.sh]: Why is this needed?
This wont be merged into OpenWRT unless a really good and documented reason.
Since other devices use the same script it should work without this

Comment (7604timo) [reply package/base-files/files/lib/functions/system.sh]: It didn't work on my device, and it turned out to be the shell interpreter. Specifically this part:
${nic//:/}
Was causing an error and not adding to the mac address as expected. That's all I know..

I'm new at this, so it's entirely possible I was doing something wrong however..

Tim

Comment (robimarko) [reply package/base-files/files/lib/functions/system.sh]: That is weird if the same function works for other devices

Comment (7604timo) [reply package/base-files/files/lib/functions/system.sh]: I was running it on my initramfs image, and I put in extra debug statements that echo'd out to a log file. That got me the error. It could be something to do with the way I was running it all, but the fix I made made it all work for me.

Comment (robimarko) [reply package/base-files/files/lib/functions/system.sh]: Can you further test it since your fix changes base files it must be really explained why and what since that could break all other devices

Comment (7604timo) [reply package/base-files/files/lib/functions/system.sh]: I think I've broken my github again.

Suddenly it thinks I want to merge another 42 commits to yours, and the WAC510 profile isn't in my 'make menuconfig' anymore.

Comment (7604timo) [reply package/base-files/files/lib/functions/system.sh]: I'm starting again..
